### ISSUE TITLE
Contrast too low

### ISSUE THREAD

**Issue by JEThompson97** [UserOfficeProject/stfc-user-office-project#113]
Part of #68
The contrast between text and background colour is too low in several instances throughout the app.

_Note_: contrast depends on colour, but also _size_ of the foreground (i.e. if the text is bigger, it poses less of a contrast issue)

#### Examples include:
(as identified by the [WAVE tool](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh))

![image](https://user-images.githubusercontent.com/31534888/117136656-94567a00-ada0-11eb-9639-9528bc9d9cbb.png)

![image](https://user-images.githubusercontent.com/31534888/117309587-778e7500-ae7a-11eb-9eda-35bbf173e387.png)

![image](https://user-images.githubusercontent.com/31534888/117136672-991b2e00-ada0-11eb-812e-34cef04de2ec.png)

![image](https://user-images.githubusercontent.com/31534888/117310368-203cd480-ae7b-11eb-8893-d6c492919cfd.png)

![image](https://user-images.githubusercontent.com/31534888/117145103-a9380b00-adaa-11eb-9780-bbf2fdd7b09e.png)

![image](https://user-images.githubusercontent.com/31534888/117310485-38acef00-ae7b-11eb-9e18-29f61708a0d6.png)

![image](https://user-images.githubusercontent.com/31534888/117309534-67769580-ae7a-11eb-84f6-0dd225808430.png)

/overwrite opened 2020-09-04

**Comment by JEThompson97** [comment]
A follow up assessment should be conducted following #69

**Comment by TickleThePanda** [comment]
@simonfernandes - will this be fixed by #69

**Comment by simonfernandes** [comment]
@TickleThePanda I think mostly, though as Edward mentioned we'll still need to do an assessment. One of the areas that I know hasn't been changed and will still need fixing is this grey on white/grey text:

![image](https://user-images.githubusercontent.com/31534888/117309534-67769580-ae7a-11eb-84f6-0dd225808430.png)

**Comment by TickleThePanda** [comment]
Closed by #113

**Comment by TeraMike** [comment]
Reopening this issue because there are a few contrast issues I missed due to one of my roles not working properly, including the example in Simon's comment on 30/6.

**Comment by TeraMike** [comment]
Closed by https://github.com/UserOfficeProject/user-office-frontend/pull/525
